docs/source/getting_started/tutorials/ecms_tutorial.rst

```diff
@@ -0,0 +1,1054 @@
+
+
+Alternatively, if we calculate the calibration factors in the same
+``ixdat`` session as our data treatment, we can also directly apply the
+calibration without having to create the calibration object first. In
+this way, we can also make sure that we use a specific calibration
+factor if we have determined several for one analyte.
+
+.. code:: ipython3
+
+    # Lets import the file again, so we can be sure there's no calibration attached yet
+    good_bit2 = Measurement.read(
+        "the_good_bit.csv", reader="ixdat"
+    )
+    # Add the EC calibration first
+    good_bit2.calibrate(RE_vs_RHE=0, A_el=0.197)
+    
+    # Let's check what MS calibrations we have saved in our list
+    print(cal_results)
+
+
+.. parsed-literal::
+
+    skipping the following line:
+    ixdat version = 0.2.5
+    
+    skipping the following line:
+    backend_name = BackendBase(none, address=none)
+    
+    [MSCalResult(name=H2@M2, mol=H2, mass=M2, F=0.3378605863775754), MSCalResult(name=O2@M32, mol=O2, mass=M32, F=0.01182634523322349), MSCalResult(name=H2_M2_gas, mol=H2, mass=M2, F=0.3191356304426369), MSCalResult(name=O2_M32_gas, mol=O2, mass=M32, F=0.09198714869342346), MSCalResult(name=C2H4@M26, mol=C2H4, mass=M26, F=0.08057166014411674)]
+    
+
+.. code:: ipython3
+
+    # Let's calibrate using the EC-MS calibration for hydrogen and the gas calibration for oxygen
+    good_bit2.calibrate(ms_cal_results=[cal_results[0], cal_results[3]])
+    
+    # Check the calibration most recently added to the data
+    print(good_bit2.calibration_list[0].ms_cal_results)
+    
+    # And plot it once again:
+    good_bit2.plot_measurement(mol_list=["H2", "O2"])
+
+
+.. parsed-literal::
+
+    [MSCalResult(name=H2@M2, mol=H2, mass=M2, F=0.3378605863775754), MSCalResult(name=O2_M32_gas, mol=O2, mass=M32, F=0.09198714869342346)]
+    
+
+
+
+.. parsed-literal::
+
+    [<AxesSubplot:xlabel='time / [s]', ylabel='signal / [mol/s]'>,
+     <AxesSubplot:xlabel='time / [s]', ylabel='$U_{RHE}$ / [V]'>,
+     None,
+     <AxesSubplot:ylabel='J / [mA cm$^{-2}$]'>]
+
+
+
+
+.. image:: EC-MS_ixdat_tutorial_files/EC-MS_ixdat_tutorial_54_2.png
+
+
+Plotting calibrated EC-MS data
+==============================
+
+Now that we’ve calibrated the data, we want to make some nice plots. In
+the sections below, we will explore some ways of using ixdat’s plotter
+and modify the plots using matplotlib.
+
+EC-MS plot calibrated gases using two y axes
+--------------------------------------------
+
+.. code:: ipython3
+
+    meas_a = good_bit.cut(tspan=[5035, 5200]) #cut the data to the interesting section
+    meas_a.tstamp += meas_a.t[0]  #to get the figure showing the time starting at 0
+    
+    axes_a = meas_a.plot_measurement(
+        mol_lists=[
+            ["H2"],  # left y-axis
+            ["O2"],  # right
+        ],
+        tspan_bg=[
+            None,
+            [0, 20],
+        ],  # [left, right] y-axes
+        unit=["nmol/s", "pmol/s/cm^2"],  # [left, right] y-axes
```

Comment:
It's fine demo of the code, but wierd, I think, to have H2 and O2 in different units in this case. Maybe save different left and right units for a time where one axis plots a carrier gas?